src/FSharp.Build/Fsc.fs

```diff
@@ -345,6 +347,8 @@ type public Fsc() as this =
         if deterministic then
             builder.AppendSwitch("--deterministic+")
 
+        builder.AppendOptionalSwitch("--parallelcompilation", parallelCompilation)
+
```

Comment:
Thinking out loud - this can probably fail for the same reason we had `realsig` failing in VS and not in SDK. Newer FSharp.Build, older SDK/Compiler?

Reply:
This is why it uses the AppendOptionalSwitch instead of AppendSwitch, and it is an optional bool.
RealSig was using a nonoptional bool before, now it is also redone to be safe.

But thanks for keeping an eye on this.